tests/Doctrine/Tests/DBAL/Types/ArrayTest.php

```diff
@@ -52,7 +46,6 @@ public function testArrayConvertsToPHPValue()
 
     public function testConversionFailure()
     {
-        error_reporting( (E_ALL | E_STRICT) - \E_NOTICE );
```

Comment:
Then you should probably also delete the now-obsolete imports at the top of the file:

```php
use const E_ALL;
use const E_STRICT;
use function error_reporting;
```

Also, there's the same logic in file `tests/Doctrine/Tests/DBAL/Types/ObjectTest.php` (from the same original commit 2cc1cdc56d7dc09445ca935ce59b5a2755699302)

Reply:
@guilliamxavier you are right. I will make the changes you suggested. However I am not sure what the maintainers think about this PR.